physical/consul.go

```diff
@@ -5,9 +5,12 @@ import (
 	"sort"
 	"strings"
 	"time"
+	"net/http"
+	"errors"
 
 	"github.com/armon/go-metrics"
 	"github.com/hashicorp/consul/api"
+	"github.com/hashicorp/consul/tlsutil"
```

Comment:
We should not be pulling in this library from Consul, that is more of an internal to Consul thing. We don't need any of that functionality anyways, since we can probably just use `tls.LoadX509KeyPair`